terraform/deployments/cluster-infrastructure/ckan_iam.tf

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Seems we don't have any module which is a great fit for this at the moment. It's not cluster infrastructure because it's app-specific, but it's also not really govuk-publishing-infrastructure either.

For now I'd probably go with govuk-publishing-infrastructure (assuming you'd rather avoid the faff of introducing a new root module in this PR) but at some point in the near future we probably want a `data-gov-uk-infrastructure` module. What do you reckon?

Reply:
I've added a new root module like you suggested, it does make sense to keep the DGU resources isolated from the main stack.